src/mainwindow.cpp

```diff
@@ -132,11 +133,15 @@ MainWindow::MainWindow(QWidget *parent) :
     ui->mainPageStack->setCurrentWidget(ui->startPage);
     ui->openFileButton->setFocus();
 
+    auto costDelegate = new CostDelegate(this);
+
     auto bottomUpCostModel = new BottomUpModel(this);
     setupTreeView(ui->bottomUpTreeView,  ui->bottomUpSearch, bottomUpCostModel);
+    ui->bottomUpTreeView->setItemDelegate(costDelegate);
```

Comment:
uhm, don't we only want to set that for the cost columns? This sets it for all columns - is that correct? I don't think so.